openhands/core/config/__init__.py

```diff
@@ -0,0 +1,39 @@
+from .agent_config import AgentConfig
+from .app_config import AppConfig
+from .config_functions import (
+    finalize_config,
+    get_llm_config_arg,
+    get_parser,
+    load_app_config,
+    load_from_env,
+    load_from_toml,
+    parse_arguments,
+)
+from .config_utils import (
+    OH_DEFAULT_AGENT,
+    OH_MAX_ITERATIONS,
+    UndefinedString,
+    get_field_info,
+)
+from .llm_config import LLMConfig
+from .sandbox_config import SandboxConfig
+from .security_config import SecurityConfig
```

Comment:
IIRC we have settled to use absolute imports in the past: https://github.com/All-Hands-AI/OpenHands/issues/3491

Reply:
Yeh, but it's not recommended and if this folder should be moved elsewhere the relative notation would still work.